### PR TITLE
Only push to NuGet for Full Build (cron)

### PR DESCRIPTION
**Pull request by TravisEz13**
Currently, every merge is also publishing.

### CODE REVIEW

File: tools/travis.ps1 [hunk 1]
```diff
@@ -181,7 +181,11 @@ else
         $packages += Start-PSPackage  @packageParams -Type AppImage
         foreach($package in $packages)
         {
-            if($env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
+            # Publish the packages to the nuget feed if:
+            # 1 - It's a fullBuild (allready checked, for not a PR)
```
Comment (adityapatwardhan) [inline tools/travis.ps1]: Typo in allready -> already

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]
I found another issue, we seem to be updating the badge on full build, including the one with [Feature]. We need a way to identify a CRON build and then do nuget publish and badge update.

File: tools/travis.ps1 [hunk 2]
```diff
@@ -181,7 +181,11 @@ else
...
+            # 2 - We have the info to publish (NUGET_KEY and NUGET_URL)
+            # 3 - it's a nupkg file
+            if($isFullBuild -and $env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
```
Comment (adityapatwardhan) [inline tools/travis.ps1]: This also published if it is [Feature] run?

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: tools/travis.ps1 (hunk 2 above)
Comment (TravisEz13) [inline tools/travis.ps1]: Only if we merge a feature... that is equivalent to a daily build, I'm fine with it.  If it's a PR, it won't publish.   And if you squash merge, you will likely, remove the `[Feature]` from the commit message.

Comment (TravisEz13) [inline tools/travis.ps1]: actually, a true merge would change the description of the last commit message too.  Only merging to master using the rebase option would have this possible effect.

**Comment by TravisEz13** [comment]
The badge issue is being fixed in #4536

File: tools/travis.ps1 [hunk 3]
```diff
@@ -181,7 +181,11 @@ else
         $packages += Start-PSPackage  @packageParams -Type AppImage
         foreach($package in $packages)
         {
-            if($env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
+            # Publish the packages to the nuget feed if:
+            # 1 - It's a fullBuild (already checked, for not a PR)
+            # 2 - We have the info to publish (NUGET_KEY and NUGET_URL)
+            # 3 - it's a nupkg file
+            if($isFullBuild -and $env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
```
Comment (daxian-dbw) [inline tools/travis.ps1]: After merging a PR to master, if the commit message happen to have `[Feature]` tag, we will publish the nuget package as well. Is that desired? Maybe we should use `$isDailyBuild` instead.

Comment (TravisEz13) [inline tools/travis.ps1]: Rebased, and updated.

**Review by daxian-dbw — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Pull request merged**